backend/tests/viewAppointmentsController.test.js

```diff
@@ -390,6 +390,293 @@ describe('View appointments', () => {
     })
   })
 
+  describe('when there are video link appointments with pre and post appointments within the same booking', () => {
+    beforeEach(() => {
+      whereaboutsApi.getAppointments.mockReturnValue([
```

Comment:
This is quite a large test - it could do with splitting up if we can. I suggest one test for each type of VLB - assuming they are testing different things.